tequilapi/http_routes.go

```diff
@@ -2,16 +2,21 @@ package tequilapi
 
 import (
 	"github.com/julienschmidt/httprouter"
+	"github.com/mysterium/node/ipify"
 	"github.com/mysterium/node/tequilapi/endpoints"
 	"os"
 	"time"
 )
 
+// NewAPIRouter returns new api router with status endpoints
 func NewAPIRouter() *httprouter.Router {
 	router := httprouter.New()
 	router.HandleMethodNotAllowed = true
 
 	router.GET("/healthcheck", endpoints.HealthCheckEndpointFactory(time.Now, os.Getpid).HealthCheck)
 
+	ipifyClient := ipify.NewClient()
+	router.GET("/ip", endpoints.NewCurrentIPEndpoint(ipifyClient.GetPublicIP).CurrentIP)
```

Comment:
It should go somewhere under `/v1..`

Reply:
Maybe `GET /v1/connection/ip` is right place

Reply:
We don't have `/v1`, but it makes sense to move it inside `connection`.

Reply:
moved to `connection`